itests/hive-unit/src/test/java/org/apache/hive/jdbc/TestJdbcWithMiniHS2.java

```diff
@@ -1098,11 +1098,11 @@ public void testHttpRetryOnServerIdleTimeout() throws Exception {
     String userName = System.getProperty("user.name");
     Connection conn = getConnection(miniHS2.getJdbcURL(testDbName), userName, "password");
     Statement stmt = conn.createStatement();
-    stmt.execute("select from_unixtime(unix_timestamp())");
+    stmt.execute("select from_unixtime(to_unix_timestamp())");
     // Sleep for longer than server's idletimeout and execute a query
     TimeUnit.SECONDS.sleep(10);
     try {
-      stmt.execute("select from_unixtime(unix_timestamp())");
+      stmt.execute("select from_unixtime(to_unix_timestamp())");
```

Comment:
this method call looks odd - `to_unix_timestamp` looks like a conversion function ; but without arguments its just odd - I think we should keep `unix_timestamp` as an alias